api2/cron-task/storePeggedPrices.ts

```diff
@@ -0,0 +1,39 @@
+import dynamodb, { getHistoricalValues } from "../../src/utils/shared/dynamodb";
+import peggedAssets from "../../src/peggedData/peggedData";
+import { getPrices } from "../../src/adapters/peggedAssets/prices";
+import { store } from "../../src/utils/s3";
+import { getTimestampAtStartOfDay } from "../../src/utils/date";
+import {
+  dailyPeggedPrices,
+  hourlyPeggedPrices,
+} from "../../src/peggedAssets/utils/getLastRecord";
+import { bridgeInfo } from "../../src/peggedData/bridgeData";
+import { getCurrentUnixTimestamp } from "../../src/utils/date";
+import { cache } from "../cache";
+
+export default async function handler() {
+  const prices = await getPrices(peggedAssets)
+  cache.peggedPrices = prices
+  if (!cache.historicalPrices) cache.historicalPrices = []
+
+  // store bridge info (name, links) in s3
+  await store("bridgeInfo.json", JSON.stringify(bridgeInfo))
+
+  const hourTimestamp = getCurrentUnixTimestamp()
+  const dayTimestamp = getTimestampAtStartOfDay(hourTimestamp)
+
+  // store hourly prices in db
+  await dynamodb.put({ PK: hourlyPeggedPrices, SK: hourTimestamp, prices: prices, })
+  await dynamodb.put({ PK: dailyPeggedPrices, SK: dayTimestamp, prices: prices, })
```

Comment:
isnt this overwriting daily prices all the time?

Reply:
yeah, it is intentional. dont think it is an issue.

Reply:
its okay but afaik before we werent overwriting so previous daily data will be at the start of the day while new data will be at the end of the day, so there will be one day with 48h difference in data points